src/main/java/exceptions/MangaDoesNotExistException.java

```diff
@@ -0,0 +1,12 @@
+package exceptions;
+
+public class MangaDoesNotExistException extends TantouException {
+    public MangaDoesNotExistException(String mangaName) {
+        super(generateMangaDoesNotExistMessage(mangaName));
+    }
+
+    private static String generateMangaDoesNotExistMessage(String mangaName) {
+        return "Manga [" + mangaName + "] does not exist! "
+                + "Manga needs to exist in our catalog first before adding their sales data!";
+    }
```

Comment:
See above comment:
>I appreciate the custom exception for this particular invalid input, but I think we should make this message more generic in case other Command classes want to use it.

Reply:
Fixed in #106